modules/ROOT/pages/sharepoint/sharepoint-connector.adoc

```diff
@@ -157,17 +245,17 @@ To build and run this demo project, you need:
 * Anypoint Studio with at least the Mule 4.0 Runtime.
 * Microsoft SharePoint Connector v2.0.0 or higher.
 
-=== To Test the Flow
+=== Test the Flow
 
-. Import the demo project into your workspace using Anypoint Exchange or using the Import command in the File menu.
+. Import the demo project into your workspace using Anypoint Exchange or using the *File* > *Import command.
 . Specify your credentials in the `/src/main/app/mule-app.properties` file:
 . Run the project in Studio.
 . Type `+localhost:8081/demo+` in your browser to access the selection menu of the demo.
-. Optionally you can configure the Connection Timeout and Read Timeout.
-The Connection Timeout is the timeout in making the initial connection with the server.
-The Read Timeout is the timeout on waiting to read data from the server.
+. Optionally you can configure *Connection Timeout* and *Read Timeout*.
+*Connection Timeout* is the timeout in making the initial connection with the server.
+*Read Timeout* is the timeout on waiting to read data from the server.
```

Comment:
Instead of defining "timeout" with "timeout," consider...:

*Connection Timeout* is the maximum time allowed to make the initial server connection.
*Read Timeout* is the maximum time allowed to read data from the server.